src/init.cpp

```diff
@@ -994,6 +995,12 @@ bool AppInitParameterInteraction()
         }
     }
 
+    // Basic filters index must be enabled to serve compact filters
```

Comment:
nit:
```suggestion
    // Basic filters index must be enabled to serve compact filters
    // because it is the only available filter option for now
```

Reply:
done